packages/react/src/React.js

```diff
@@ -83,6 +85,10 @@ const React = {
   createElement: __DEV__ ? createElementWithValidation : createElement,
   cloneElement: __DEV__ ? cloneElementWithValidation : cloneElement,
   createFactory: __DEV__ ? createFactoryWithValidation : createFactory,
+  jsx: __DEV__ ? jsxWithValidation.bind(null, false) : jsx,
```

Comment:
Can you add a feature flag in ReactFeatureFlags for this and then only conditionally add these if the flag is on? Like `enableStableConcurrentModeAPIs` is doing below.

It allows us to release from master once it lands.

You might have to find other ReactFeatureFlags forked files and it there too. E.g. you can turn it on in the ReactFeatureFlags.www.js